tests/ocf-url_link_path_absolute/EPUB/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-03-01</dc:date>
+    <dc:description>Use a path-absolute link from the content to a photograph. The folder hierarchy containing the photograph starts at the root level.</dc:description>
+    <dc:identifier id="pub-id">ocf-url_link_path_absolute</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Path-absolute links</dc:title>
```

Comment:
Perhaps "Absolute image reference with leading slash"?

Reply:
I prefer to keep the official term, which is used in the spec.